build-system/common/utils.js

```diff
@@ -146,9 +146,8 @@ function getFilesToCheck(globs, options = {}, ignoreFile = undefined) {
       return [];
     }
     // forbidden-terms always needs to be checked.
-    if (!filesChanged.includes('build-system/test-configs/forbidden-terms.js')) {
-      filesChanged.push('build-system/test-configs/forbidden-terms.js');
-    }
+    const forbiddenTerms = 'build-system/test-configs/forbidden-terms.js';
+    filesChanged.includes(forbiddenTerms) && filesChanged.push(forbiddenTerms);
```

Comment:
Rewriting it this way is harder to read imo and masks that this change introduced a bug (dropping the `!` so it _only_ pushes if already present)

Reply:
Good catch on the missing `!` 👏 
I also think the `if` statement reads nicer so I'm going to revert this suggestion

Reply:
Agree, good point and good catch on the missing `!`. Maybe compromise by capturing the filename in a `const` and then using it in the `if` statement?